Adyen/Constants/ClientConfig.cs

```diff
@@ -8,6 +8,7 @@ public class ClientConfig
         public static string CloudApiEndPointEULive = "https://terminal-api-live.adyen.com";
         public static string CloudApiEndPointAULive = "https://terminal-api-live-au.adyen.com";
         public static string CloudApiEndPointUSLive = "https://terminal-api-live-us.adyen.com";
+        public static string CloudApiEndPointAPSELive = "https://terminal-api-live-apse.adyen.com";
```

Comment:
General question: How do we expect our merchants to be aware of these public fields? How would they when to use them and how to find them?

Reply:
Good question, there is an example on the README. Then selecting ClientConfig shows them the different options. That should be enough right or?